perses/annihilation/relative.py

```diff
@@ -71,7 +71,8 @@ def __init__(self,
                  softcore_LJ_v2_alpha = 0.85,
                  softcore_electrostatics_alpha = 0.3,
                  softcore_sigma_Q = 1.0,
-                 interpolate_old_and_new_14s = False):
+                 interpolate_old_and_new_14s = False,
+                 omitted_terms = None):
```

Comment:
added `omitted_terms` for valence terms that can be annealed _in_ over a protocol; these are generated by the `FFAllAngleGeometryEngine`, but in the pipeline, i am leaving this as `None` for the time being since we do not want to anneal terms in...this will be handled in a future PR